### ISSUE TITLE
COMETS 1.4.  When a metabolite is adjusted for, its array position is shown in adjvars instead of its name

### ISSUE THREAD

**Issue by steven-moore** [CBIIT/R-cometsAnalytics#71]
If adjusting for a metabolite, the name is not shown onscreen.  Similar to the issue #70 .

![image](https://user-images.githubusercontent.com/34717005/49756852-f3c00680-fc88-11e8-81f2-eb417cb45b9d.png)

**Comment by Mathelab** [comment]
I'm assuming here that we can remove the adjvars column?
Ewy

**Comment by steven-moore** [comment]
We want to keep the adjvars column, so that investigators can see which metabolite they adjusted for.

For example, if I was testing for the relationship between metabolites and diabetes, I might want to adjust for glucose from the metabolomics data.  This would allow me to see if the metabolites have predictive power above and beyond their correlation with glucose.  Retaining the adjvars and showing “glucose” there allows me to track which models were and were not glucose adjusted.

S

**Comment by Mathelab** [comment]
Yes of course, makes sense.
Ewy

**Comment by Mathelab** [comment]
Hi everyone,

I have pushed changes. Variables adjvars and adjspec are redundant.  I have kept both in because I want to make sure one is not used downstream.  Will clean this up later.

Best,
Ewy

From: "Mathe, Ewy" <Ewy.Mathe@osumc.edu>
Date: Monday, December 10, 2018 at 5:26 PM
To: CBIIT/R-cometsAnalytics <reply@reply.github.com>, CBIIT/R-cometsAnalytics <R-cometsAnalytics@noreply.github.com>
Cc: Comment <comment@noreply.github.com>
Subject: Re: [CBIIT/R-cometsAnalytics] COMETS 1.4. When a metabolite is adjusted for, its array position is shown in adjvars instead of its name (#71)

The adjvars and adjspec columns are the same, but one is the right value, so I’m removing adjvars.  Adjspec is what the user has input so we’re OK.

Going to push shortly.

Best,
Ewy

From: "Mathe, Ewy" <Ewy.Mathe@osumc.edu>
Date: Monday, December 10, 2018 at 5:16 PM
To: CBIIT/R-cometsAnalytics <reply@reply.github.com>, CBIIT/R-cometsAnalytics <R-cometsAnalytics@noreply.github.com>
Cc: Comment <comment@noreply.github.com>
Subject: Re: [CBIIT/R-cometsAnalytics] COMETS 1.4. When a metabolite is adjusted for, its array position is shown in adjvars instead of its name (#71)

Yes of course, makes sense.
Ewy
